### ISSUE TITLE
`MaximumLineLength` and `MaxLineLength` are not working properly

### ISSUE THREAD

**Issue by GrzegorzBobryk** [detekt/detekt#7551]
## Expected Behavior

Set `maxLineLength` should be the same for every line in kotlin file. I also need [only one of those objects](https://detekt.dev/docs/rules/formatting/#maximumlinelength) set to active to verification work properly.

## Observed Behavior

When I remove `MaximumLineLength` or `MaxLineLength` then detekt is throwing error in [adapter](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/sample/src/main/kotlin/pl/beavercoding/viewbindingdelegate/ExampleAdapter.kt). It is seeing problem in method `onCreateViewHolder` despite that this method is 122 characters long and [both of those properties](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/config/detekt.yml) are set on 130 lines. It is working fine if both of those objects are present but after removing one of them it is crashing.

## Steps to Reproduce

**first approach:**
1. Comment out `MaximumLineLength` in [yaml](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/config/detekt.yml);
2. Run command:
```shell
./gradlew detektMain detektTest
```
or
```shell
./gradlew detekt  
```
You will receive
> ViewBindingDelegate\sample\src\main\kotlin\pl\beavercoding\viewbindingdelegate\ExampleAdapter.kt:16:1: Exceeded max line length (120) [MaximumLineLength]

**second approach:**
1. Reset changes in [yaml](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/config/detekt.yml);
2. Comment out `MaxLineLength` in [yaml](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/config/detekt.yml);
3. Run command 
```shell
./gradlew detektMain detektTest
```
or
```shell
./gradlew detekt  
```
You will receive
> ViewBindingDelegate\sample\src\main\kotlin\pl\beavercoding\viewbindingdelegate\ExampleAdapter.kt:16:1: Exceeded max line length (120) [MaximumLineLength]

## Context

I am trying to have single source of truth and [documentation ](https://detekt.dev/docs/rules/formatting/#maximumlinelength) is suggesting the same. I want to not being forced on having two settings for the same thing.

## Your Environment

* Version of detekt used: 1.23.6
* Version of Gradle used (if applicable): 8.7
* Gradle scan link (add `--scan` option when running the gradle task): I do not want to show my ip adress
* Operating System and version: Windows 11
* Link to your project (if it's a public repository): Two links added in description above. In case you miss it I am pasting it once again:
  - [adapter](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/sample/src/main/kotlin/pl/beavercoding/viewbindingdelegate/ExampleAdapter.kt);
  - [yaml](https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/master/config/detekt.yml).

**Comment by 3flex** [comment]
What's the bug?

Only one rule should be enabled. If they're both enabled, set them to the same line length limit.

Some rules are enabled by default so you will need to set `active: false` one one of them.

**Comment by GrzegorzBobryk** [comment]
If I have only one of this rules I am getting:
> ViewBindingDelegate\sample\src\main\kotlin\pl\beavercoding\viewbindingdelegate\ExampleAdapter.kt:16:1: Exceeded max line length (120) [MaximumLineLength]

This line have 122 characters, so it is 2 characters longer than default.

Both of this values are set to `130`

If I have both of them it works fine. The funny part is that it does not matter witch one you will comment both will crash on the same line.

Bug is that:
- I cannot have only one because detekt will not work correctly;
- If I am setting 130 on `MaximumLineLength` and `MaxLineLength` is not present in yaml it sees problem with line that have 122 characters;
- If I am setting 130 on `MaxLineLength` and `MaximumLineLength` is not present in yaml  it sees problem with line that have 122 characters.

Both of this rules have problem with the same specific line.

**Comment by cortinico** [comment]
Can you share the line you're mentioning the issue is reported for: `ViewBindingDelegate\sample\src\main\kotlin\pl\beavercoding\viewbindingdelegate\ExampleAdapter.kt:16:1`

> If I have both of them it works fine. The funny part is that it does not matter witch one you will comment both will crash on the same line.

FYI this is potentially a false-positive and not a crash as you mentioned

**Comment by GrzegorzBobryk** [comment]
Yes, it is false-positive. I have used wrong words to describe that. 

The line:
https://github.com/GrzegorzBobryk/ViewBindingDelegate/blob/fa73daff1721fc8f4b7418f66361caecb22b4ca9/sample/src/main/kotlin/pl/beavercoding/viewbindingdelegate/ExampleAdapter.kt#L16C1-L16C123

The false-positive:
`ViewBindingDelegate\sample\src\main\kotlin\pl\beavercoding\viewbindingdelegate\ExampleAdapter.kt:16:1: Exceeded max line length (120) [MaximumLineLength]`
